### ISSUE TITLE
NuGet Packages out of date

### ISSUE THREAD

**Issue by AshotN** [kunzmi/managedCuda#61]
Either I'm missing something, or all the packages on NuGet are out of date. Is there any plans on updating them?

**Comment by kunzmi** [comment]
Hi, unfortunately you haven't missed anything… I just started a new job, moved to another country and am still in the process to get settled here :) Once I have a free weekend or some other spare time, I'll update the packages!

**Comment by AshotN** [comment]
Alright cool, I got it all working with NuGet anyway. Thank you for your work!

**Comment by BradJeong** [comment]
ManagedCudaSamples has assembly load problem.

**Comment by kunzmi** [comment]
Cuda 10 is now available as Nuget! (--> Assembly load problem: likely x64/x86 version mismatch of a library...)

**Comment by STeeL835** [comment]
> Cuda 10 is now available as Nuget! 

I have tried to run a sample `vectorAdd_nvrtc` (I have CUDA 10.0 and ran sample with nuget packages) but it was throwing an exception `Coudn't find a library nvrtc64_100`. I went to CUDA Toolkit folder and for some reason there was not nvrtc64_100.dll, but nvrtc64_100<b>_0</b>.dll. Making a copy of the dll but without `_0` in it's name worked.
If it's not just me, maybe this should be fixed in code

**Comment by kunzmi** [comment]
Yes indeed, the nvrtc lib isn't named like the others. As there were no changes in the API I didn't run any tests, hence missed it.
I'll fix it in code...
